e2e/src/pages/profile.po.ts

```diff
@@ -0,0 +1,82 @@
+import { $, $$ } from 'protractor';
+import { BaseComponent } from './base.component';
+
+export class ProfilePo extends BaseComponent {
+  readonly url = '/u/';
+
+  private readonly btnEdit = $('.btn-edit-trigger .Button-icon');
+  private readonly btnViewProfile = $('.btn-edit-trigger .fl-icon-user');
```

Comment:
or you can use one btn for turn on/off '.btn-edit-trigger:not(.ng-hide)'. As you prefer